ios/browser/api/local_data_file_service/local_data_file_service_wrapper.mm

```diff
@@ -0,0 +1,58 @@
+/* Copyright (c) 2023 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include <Foundation/Foundation.h>
+#include <sys/qos.h>
+#include <string>
+#include "url/gurl.h"
+
+#include "brave/components/brave_component_updater/browser/local_data_files_service.h"
+#include "brave/components/url_sanitizer/browser/url_sanitizer_component_installer.h"
+#include "brave/components/url_sanitizer/browser/url_sanitizer_service.h"
+#include "brave/ios/browser/api/local_data_file_service/local_data_file_service_installer_delegate.h"
+#include "brave/ios/browser/api/local_data_file_service/local_data_file_service_wrapper+private.h"
+#import "net/base/mac/url_conversions.h"
+
+using brave::URLSanitizerComponentInstaller;
+using brave::URLSanitizerService;
+using brave_component_updater::LocalDataFilesService;
+using brave_component_updater::LocalDataFilesServiceFactory;
+using local_data_file_service::LocalDataFileServiceDelegate;
+
+// LocalDataFileServiceWrapper
+@interface LocalDataFileServiceWrapper () {
+  std::unique_ptr<LocalDataFileServiceDelegate> _delegate;
```

Comment:
This is not allowed in C++. Likewise for all the variables below it.
Variable names `cannot` begin with an `underscore`.

> **17.4.3.1.2 Global names [lib.global.names]
> Certain sets of names and function signatures are always reserved to the implementation:**
> 
> Each name that contains a double underscore (__) or begins with an underscore followed by an uppercase letter (2.11) is reserved to the implementation for any use.
>
> Each name that begins with an underscore is reserved to the implementation for use as a name in the global namespace.165
> 165) 
>
> Such names are also reserved in namespace ::std (17.4.3.1).